versions/go.tex

```diff
@@ -591,6 +589,31 @@ \subsection*{Geschäftsordnungsanträge}\label{geschuxe4ftsordnungsantruxe4ge-1}
 Formale Gegenrede bedeutet nur bekanntzugeben, dass man dagegen ist,
 inhaltliche Gegenrede beinhaltet eine Begründung.
 
+\subsubsection*{Teilnamentliche und Geheime Abstimmung}
+
+Normale Abstimmungen auf der ZaPF sind nicht geheim, aber das
+Abstimmungsverhalten wird auch nicht protokolliert und nur anwesende Teilnehmika
+der ZaPF können sehen ob und wie andere Fachschaften abgestimmt haben.
+Die Teilnamentliche Abstimmung soll dazu dienen einen Antrag besonders zu
+unterstreichen indem man ihm mit dem eignen Namen mehr Gewicht verleiht.
+Durch die Bekanntgabe der Intention namentlich abzustimmen können andere Fachschaften
+abschätzen, ob durch die teilnamentliche Abstimmung ihr Abstimmverhalten bestimmbar
+wird und ggf. Gegenrede einlegen. Dies setzt voraus, dass nur Fachschaften namentlich
```

Comment:
Das ist leider nicht der Fall, da es keine Verknüpfung der Absichtserklärung und der tatsächlichen namentlichen Abstimmung gibt. Es kann nie davon ausgegangen werden, dass alle Fachschaften, die planen, namentlich abzustimmen, dies auch vorher ankündigen. Daher muss eine Fachschaft immer Gegenrede einlegen wenn sie der Anonymität ihrer Stimme im Protokoll sicher sein möchte.

Der nächste Satz legt zwar nahe, dass dieses Problem durchaus Bewusst ist. Ich frage mich dann aber, warum dieses Problem wissentlich in Kauf genommen wird. Hier muss also mindestens noch eine Änderung am Antrag erfolgen um die Integrität der demokratischen Prozesse des Plenums zu sichern.